packages/react-query/src/BalanceVoting.tsx

```diff
@@ -13,19 +14,20 @@ import FormatBalance from './FormatBalance.js';
 interface Props {
   children?: React.ReactNode;
   className?: string;
+  isReferenda?: boolean;
   label?: React.ReactNode;
   params?: AccountId | AccountIndex | Address | string | Uint8Array | null;
 }
 
-function BalanceVoting ({ children, className = '', label, params }: Props): React.ReactElement<Props> {
+function BalanceVoting ({ children, className = '', isReferenda, label, params }: Props): React.ReactElement<Props> {
   const { api } = useApi();
   const allBalances = useCall<DeriveBalancesAll>(api.derive.balances?.all, [params]);
 
   return (
     <FormatBalance
       className={className}
       label={label}
-      value={allBalances?.votingBalance}
+      value={isReferenda && api.query.convictionVoting ? allBalances?.votingBalance.add(allBalances ? allBalances.reservedBalance : new BN(0)) : allBalances?.votingBalance}
```

Comment:
```suggestion
      value={isReferenda && api.query.convictionVoting ? allBalances?.votingBalance.add(allBalances.reservedBalance) : allBalances.votingBalance}
```
Simmilar to above

Reply:
Since the useCall returns `<T> | undefined`